src/valora-dapp-list.json

```diff
@@ -313,6 +313,15 @@
       "url": "https://app.ubeswap.org/",
       "listOnAndroid": true,
       "listOnIos": true
+    },
+    {
+      "name": "Revoke Cash",
+      "id": "revokecash",
+      "categoryId": "finance-tools",
+      "categories": ["finance-tools"],
+      "url": "https://revoke.cash/address/{{address}}?chainId=42220",
```

Comment:
What do you think about removing the chainId query parameter? I'd rather go with the revoke.cash default for multichain support.

Reply:
We have something similar with Toucan Protocol. I think we should include it until we have a more robust solution for handling specifying chain in the Dapps explorer. If we omit the chain Id the user lands on a page with Ethereum  selected which is less useful to those who are not in the Beta.